src/ocm/components/clusterConfiguration/discoveryImageModal.tsx

```diff
@@ -9,30 +9,34 @@ type DiscoveryImageModalButtonProps = {
   ButtonComponent?: typeof Button | typeof ToolbarButton;
   cluster: Cluster;
   idPrefix: string;
+  isSNO: boolean;
+};
+
+type DiscoveryImageModalProps = {
```

Comment:
since you have the cluster, you can use isSno(cluster) it instead of adding a property